### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by TnT420** [zeltox/Google-Drive-PDF-Downloader#9]
A support to download view only docs as well.

Reference: https://docs.google.com

**Comment by antoinefreches** [comment]
yes, this works perfectly for pdf but I would also need it for google docs/word files. TIA
